src/components/application_manager/src/message_helper.cc

```diff
@@ -2290,25 +2290,28 @@ mobile_apis::Result::eType MessageHelper::VerifyImage(
   }
 
   std::string full_file_path;
-  if (file_name.size() > 0 && file_name[0] == '/') {
+  const std::string path_delimeter = file_system::GetPathDelimiter();
+  if (file_name.size() > 0 && !file_system::IsRelativePath(file_name)) {
     full_file_path = file_name;
   } else {
     const std::string& app_storage_folder =
         app_mngr.get_settings().app_storage_folder();
     if (!app_storage_folder.empty()) {
       // TODO(nvaganov@luxoft.com): APPLINK-11293
-      if (app_storage_folder[0] == '/') {  // absolute path
-        full_file_path = app_storage_folder + "/";
+      if (!file_system::IsRelativePath(app_storage_folder)) {  // absolute path
```

Comment:
@Kozoriz minor note:  maybe if we invert the logic if(file_system::IsRelativePath(app_storage_folder)){..}else{..} it will be more readable..